services/datamanager/builtin/builtin.go

```diff
@@ -827,3 +860,58 @@ func pollFilesystem(ctx context.Context, wg *sync.WaitGroup, captureDir string,
 		}
 	}
 }
+
+func logCaptureDirSize(ctx context.Context, captureDir string, wg *sync.WaitGroup, logger logging.Logger,
```

Comment:
Awesome. Checking that you tested this alongside capture/sync?

Reply:
Yep!